lisp/ess-inf.el

```diff
@@ -3185,8 +3188,11 @@ search path related variables."
 (defun ess-display-temp-buffer (buff)
   "Display the buffer BUFF using `temp-buffer-show-function' and respecting
 `ess-display-buffer-reuse-frames'."
-  (let ((display-buffer-reuse-frames ess-display-buffer-reuse-frames))
-    (funcall (or temp-buffer-show-function 'display-buffer) buff)))
+  (if temp-buffer-show-function
```

Comment:
Why was this change needed? Unless I miss something, the code within fboundp is not the correct application of `temp-buffer-show-function`.